app/controllers/page_views_controller.rb

```diff
@@ -18,10 +18,8 @@ def create
 
   def update
     if session_current_user_id
-      page_view = PageView.where(article_id: params[:id], user_id: session_current_user_id).last
-      # pageview is sometimes missing if failure on prior creation.
-      page_view ||= PageView.create(user_id: session_current_user_id, article_id: params[:id])
-      page_view.update_column(:time_tracked_in_seconds, page_view.time_tracked_in_seconds + 15)
+      page_view = PageView.find_or_create_by(article_id: params[:id], user_id: session_current_user_id)
+      page_view.update(time_tracked_in_seconds: page_view.time_tracked_in_seconds + 15)
```

Comment:
Hello! 👋 

I'm new to this codebase, and along with looking through the code and the issues outstanding I'm also trying to take a peek at some of the PR's to get familiar with the setup. 

I just wanted to say that from my (very) unfamiliar eyes that: 

* This looks like an awesome use of #find_or_create_by and makes everything look much cleaner :) 
* Normally whenever I see `#update_column` I take it as a sign that "we want to update things without triggering callbacks". However upon circling around to `page_view.rb` I saw that, yep, the only callback is a `before_create`, so that was a nice 🐱ch as well 👍

Reply:
Welcome!!! Thanks for the PR feedback, it is always appreciated! 

Definitely feel free to dive in if you see any Issues that look interesting and you want to pick up. Anything labeled `help_wanted` or `good_first_issue` are good places to start.

Reply:
I definitely will, thanks!!